script.js

```diff
@@ -1,88 +1,69 @@
 let books = [
-    {
-   titulo: '100 años de soledad',
-   author: 'Gabriel Garcia Marquez'
-    },
-    {
-   titulo: 'El tirano',
-   author: 'Massimo Manfredy'
-    }
-]
-
-const bookList = document.querySelector('.books-display');
-let displ = ''
-books.forEach((el, index) => {    
-    displ += `
-    <div class="oneBook" id="book${index}">
-    <h4> ${el.titulo} </h4>
-    <h4> ${el.author} </h4> 
-    <button class="btn-display" id="${index}">remove</button> 
-    </div>
-    ` 
-
-});
-bookList.innerHTML = displ
-let removIds = [...document.querySelectorAll('.btn-display')].map(item => item.id)
-
-const newB = document.getElementById('bk')
-
-newB.addEventListener('submit', function(event){
+  {
+    titulo: '100 años de soledad',
+    author: 'Gabriel Garcia Marquez',
+  },
+  {
+    titulo: 'El tirano',
+    author: 'Massimo Manfredi',
+  },
+];
+
+function storage(ok) {
+  window.localStorage.setItem('localBooks', JSON.stringify(ok));
+}
```

Comment:
- [ ] kindly check the local storage after every functionality: Data is preserved in the browser's memory by using [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md), reload the page manually to check this after adding a book and removing a book.